core/src/query.rs

```diff
@@ -301,10 +301,23 @@ pub fn pop(name: &str) -> Result<String, PgmqError> {
 
 /// panics if input is invalid. otherwise does nothing.
 pub fn check_input(input: &str) -> Result<(), PgmqError> {
-    let valid = input
+    // Docs:
+    // https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS
+
+    // Default value of `NAMEDATALEN`, set in `src/include/pg_config_manual.h`
+    const NAMEDATALEN: usize = 64;
```

Comment:
Is there an API where this configuration can be read, so you don't need to guess?

Reply:
This might be close, but need to look into this a bit more. Not sure how reliable it would be for our use case.

```sql
show max_identifier_length;
 max_identifier_length 
-----------------------
 63
(1 row)
```

Reply:
https://www.postgresql.org/docs/current/runtime-config-preset.html